ferrilata-devilline/Models/Pitches.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+
+namespace ferrilata_devilline.Models
+{
+    public class Pitches
+    {
```

Comment:
I'd prefer to have this list as a field of your test class, as opposed to an object, but we can ask suvi about best practices.

Reply:
Yeah, let's ask Suvi about that.